sdk/loadtestservice/azure-developer-loadtesting/tests/test_smoke.py

```diff
@@ -8,18 +8,26 @@
 from pathlib import Path
 
 from testcase import LoadtestingTest, LoadtestingPowerShellPreparer
+from devtools_testutils import recorded_by_proxy, set_bodiless_matcher, set_custom_default_matcher
 
+test_id = os.environ.get("TEST_ID", "000")
```

Comment:
@NiveditJain you should really use powershell preparers to handle your env vars, you shouldn't be using os.environ to get your env vars. Here's an example of setup, I believe I already linked the test docs elsewhere in this pr https://github.com/Azure/azure-sdk-for-python/blob/fe6841289a45be0e89404b04332b8f7b78921c16/sdk/deviceupdate/azure-iot-deviceupdate/tests/testcase.py#L23

Reply:
updated in: https://github.com/Azure/azure-sdk-for-python/commit/5815ab611aae3a974320c78362aa37ce7c29588a